tests/e2e/util.go

```diff
@@ -7306,12 +7355,44 @@ func validate_increasedQuota(ctx context.Context, diskSize int64, totalUsedQuota
 	gomega.Expect(err).NotTo(gomega.HaveOccurred())
 	ginkgo.By(fmt.Sprintf(" quotaAfter :  %v%s", quotaAfter, string(suffix1)))
 
-	if string(suffix) == "Gi" {
-		var bytes int64 = diskSize
-		// Convert to Gi
-		diskSize = int64(bytes) / 1024
-		fmt.Printf("diskSize:  %dGi\n", diskSize)
+	sizeStr := strings.TrimSpace(diskSize)
+	ginkgo.By(fmt.Sprintf(" diskSize :  %s", diskSize))
+	// Use regex to extract the numeric part
+	rex := regexp.MustCompile(`\d+`)
+	numStr := rex.FindString(sizeStr)
+
+	// Convert to int64
+	storage, err := strconv.ParseInt(numStr, 10, 64)
+	gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+	// Determine the unit
+	diskSizeUnit := strings.ToUpper(strings.TrimPrefix(sizeStr, numStr))
 
+	if string(suffix) != string(suffix1) {
```

Comment:
Can this be a util & used across functions or test steps where required?
I see multiple duplication of this conditional check for quota size.